src/tsv.js

```diff
@@ -3,6 +3,8 @@ define([
 ], function (
     _
 ) {
+    var specialChars = /[\r\n\u2028\u2029"]/;
```

Comment:
Went through the backlog yesterday. Adding \t here should fix http://manage.dimagi.com/default.asp?104522#794142 as well correct?